### PR TITLE
CraftMyPDF (Independent Publisher)

### PR DESCRIPTION
**Pull request by troystaylor**
brave_LDfCznnnOH
brave_HqwAwjAiXh

---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

- [X] I attest that the connector works and I verified by deploying and testing all the operations. 
- [X] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [X] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [X] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [X] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [X] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by Amjed-Ayoub** [comment]
Dear partner,

We apologize for the delay you might be experiencing while we are working on a high volume of connector requests. Rest assured that we are doing everything we can to address this volume. We are taking measures like improving our processes and hiring additional team members to guarantee resolving your connector requests in a timely manner. 

Please let us know if you have any questions in the meantime.

File: independent-publisher-connectors/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,873 @@
...
+            "description": "Returns status and output file.",
+            "schema": {
+              "$ref": "#/definitions/ResponseSuccessSingleFile"
+            },
+            "headers": {}
+          }
+        }
+      }
+    },
+    "/create-async": {},
```
Comment (sriyen-msft) [inline independent-publisher-connectors/apiDefinition.swagger.json:69]: @troystaylor Please remove empty operation definitions, they would not add any value to your connector.

File: independent-publisher-connectors/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,873 @@
...
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns status and editor URL.",
+            "schema": {
+              "$ref": "#/definitions/ResponseSuccessCreateNewEditorSession"
+            },
+            "headers": {}
```
Comment (sriyen-msft) [inline independent-publisher-connectors/apiDefinition.swagger.json:289]: @troystaylor Empty property values **in the swagger** can be removed until unless you have a specific reason to keep them.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Review by sriyen-msft — APPROVED** [review]

**Comment by sriyen-msft** [comment]
@troystaylor Apologies for the delay in response. 

Congratulations! The deployment of your connector and documentation page is now complete, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program, Thanks.

**Pull request merged**